devicetypes/smartthings/zigbee-motion-detector.src/zigbee-motion-detector.groovy

```diff
@@ -101,7 +102,7 @@ def parse(String description) {
 def batteyHandler(String description){
 	def descMap = zigbee.parseDescriptionAsMap(description)
 	def map = [:]
-	if (descMap?.clusterInt == zigbee.POWER_CONFIGURATION_CLUSTER && descMap.commandInt != 0x07 && descMap.value) {
+	if (descMap?.clusterInt == zigbee.POWER_CONFIGURATION_CLUSTER && descMap.commandInt != 0x07 && descMap.value && descMap?.attrInt == 0x0021) {
```

Comment:
Did You add this condition because eWelink devices also report values of attribute 0x0020 (BatteryVoltage) ?

Reply:
Yes, if the 0x0020 voltage is reported, there will be a problem here.